docs/kfs_example.md

```diff
@@ -0,0 +1,141 @@
+
+ When using the resnet model with `grpc_predict_resnet.py`, the script processes the output from the server and displays the inference results using the previously prepared file containing labels. Inside this file, each image has an assigned number, which indicates the correct classification result.
+
+## Steps
+Clone OpenVINO&trade; Model Server GitHub repository and enter `model_server` directory.
+```
+git clone https://github.com/openvinotoolkit/model_server.git
+cd model_server
+```
+#### Download the Pretrained Model
+Download the model files and store them in the `models` directory
+```Bash
+mkdir -p models/resnet/1
+curl https://storage.openvinotoolkit.org/repositories/open_model_zoo/2021.4/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.bin https://storage.openvinotoolkit.org/repositories/open_model_zoo/2021.4/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.xml -o models/resnet/1/resnet50-binary-0001.bin -o models/resnet/1/resnet50-binary-0001.xml
+```
+
+#### Pull the Latest Model Server Image
+Pull the latest version of OpenVINO&trade; Model Server from Docker Hub :
+```Bash
+docker pull openvino/model_server:latest
+```
+
+#### Start the Model Server Container with Downloaded Model and Dynamic Batch Size
+Start the server container with the image pulled in the previous step and mount the `models` directory :
+```Bash
+docker run --rm -d -v $(pwd)/models:/models -p 9000:9000 openvino/model_server:latest --model_name resnet --model_path /models/resnet --batch_size auto --port 9000
+```
+
+#### Prepare virtualenv
+```Bash
+cd client/python/kserve-api/samples
+virtualenv .venv
+. .venv/bin/activate
+pip install -r requirements.txt
+```
+
+#### Run the Client to get metadata
+```Bash
+python3 ./kfs_grpc_model_metadata.py --grpc_port 9000 --grpc_address localhost --model_name resnet
+```
+
+#### Script Output
+```Bash
+server metadata:
+name: "resnet"
+versions: "1"
+platform: "OpenVINO"
+inputs {
+  name: "0"
+  datatype: "FP32"
+  shape: 1
+  shape: 3
+  shape: 224
+  shape: 224
+}
+outputs {
+  name: "1463"
+  datatype: "FP32"
+  shape: 1
+  shape: 1000
+}
+```
+
+#### Run the Client to perform inference
+```Bash
+python grpc_predict_resnet.py --grpc_port 9000 --images_numpy_path ../../imgs.npy --labels_numpy_path ../../lbs.npy --input_name 0 --output_name 1463 --model_name resnet --transpose_input False;
+```
+
+#### Script Output
+```Bash
+Image data range: 0.0 : 255.0
+Start processing:
+	Model name: resnet
+	Iterations: 10
+	Images numpy path: imgs.npy
+	Numpy file shape: (10, 3, 224, 224)
+
+Iteration 1; Processing time: 21.16 ms; speed 47.25 fps
+imagenet top results in a single batch:
+	 0 airliner 404 ; Correct match.
+Iteration 2; Processing time: 8.08 ms; speed 123.79 fps
+imagenet top results in a single batch:
+	 0 Arctic fox, white fox, Alopex lagopus 279 ; Correct match.
+Iteration 3; Processing time: 104.76 ms; speed 9.55 fps
```

Comment:
why one request is 10x slower?

Reply:
Probably bazel build in the meantime - rerun example with new output.